src/utils/graphql/queries/getReviewAssignment.query.ts

```diff
@@ -1,40 +1,40 @@
 import { gql } from '@apollo/client'
 
 export default gql`
-    query getReviewAssignment($reviewerId: Int!, $applicationId: Int, $stageId: Int) {
-        reviewAssignments(
-            condition: {
-                reviewerId: $reviewerId, 
-                applicationId: $applicationId
-                stageId: $stageId
-            }) {
-            nodes {
+  query getReviewAssignment($reviewerId: Int!, $applicationId: Int, $stageId: Int) {
+    reviewAssignments(
+      condition: { reviewerId: $reviewerId, applicationId: $applicationId, stageId: $stageId }
+    ) {
+      nodes {
+        id
+        applicationId
+        reviewerId
+        stageId
+        reviews {
+          nodes {
+            id
+            status
+            trigger
```

Comment:
Probably no point getting trigger here, since we have a separate hook for that, which will need to be added later.

Reply:
Done!